frontend/src/components/RegisterForm.jsx

```diff
@@ -123,6 +126,13 @@ export default function RegisterForm() {
         draggable
         pauseOnHover
       />
+      <button
+        type="button"
+        className=" m-2 text-base px-10 py-2  border rounded-lg"
+        onClick={() => navigate("/")}
+      >
+        Retour a la page d'accueil
+      </button>
```

Comment:
A link `<a>` would be more adequate and since used twice at the time to make a component